cluster/gce/config-default.sh

```diff
@@ -331,3 +331,7 @@ fi
 
 # Optional: enable certificate rotation of the kubelet certificates.
 ROTATE_CERTIFICATES="${ROTATE_CERTIFICATES:-}"
+
+# The number of services that are allowed to sync concurrently. Will be passed
+# into kube-controller-manager via `--concurrent-service-syncs`
+CONCURRENT_SERVICE_SYNCS="${CONCURRENT_SERVICE_SYNCS:-}"
```

Comment:
Could also please add this to `cluster/kubemark/gce/config-default.sh`?

Reply:
Sure, added to `cluster/kubemark/gce/config-default.sh`.